services/slam/fake/slam_test.go

```diff
@@ -0,0 +1,65 @@
+package fake
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/geo/r3"
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+	rdkutils "go.viam.com/rdk/utils"
+)
+
+func TestFakeSLAMPosition(t *testing.T) {
+	slamSvc := &FakeSLAM{Name: "test"}
+	pInFrame, err := slamSvc.Position(context.Background(), slamSvc.Name, map[string]interface{}{})
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, pInFrame.Parent(), test.ShouldEqual, slamSvc.Name)
+
+	expectedPoint := r3.Vector{X: 1.8607751801785188, Y: 34.26593374183797, Z: 0}
+	test.That(t, pInFrame.Pose().Point(), test.ShouldResemble, expectedPoint)
+
+	expectedOri := spatialmath.NewR4AA()
+	expectedOri.RZ = -1
+	expectedOri.Theta = 3.0542483867902197
+	test.That(t, pInFrame.Pose().Orientation().AxisAngles(), test.ShouldResemble, expectedOri)
```

Comment:
potential test of incrementDataCount (only increments on GetMap, if we want to keep that behavior.